course6/toyapp-adam/displaylib/sdlobject.cc

```diff
@@ -13,7 +13,7 @@ SDLObject::SDLObject()
 {
 	if(gOjectCounter++ == 0)
 	{
-		int res = SDL_Init(SDL_INIT_VIDEO);
+		auto res = SDL_Init(SDL_INIT_VIDEO);
```

Comment:
what was wrong with int?

Reply:
you're right, nothing. I just went through the codebase a couple times and opted for consistency with the usage of auto